### PR TITLE
test: resolve pr comments

### PR DESCRIPTION
**Pull request by dilin-MS**
1. Keyvault validator: Validate secret value
2. Move common function to common utility
3. Remove hardcode test account object id
4. Keyvault: Remove test account permission after validating secrets
5. Remove useless function.

### CODE REVIEW

**Review by Alive-Fish — APPROVED** [review]

File: packages/cli/tests/commonlib/keyVaultValidator.ts [hunk 1]
```diff
@@ -22,14 +25,25 @@ const baseUrlVaultSecrets = (vaultBaseUrl: string, secretName: string, secretVer
   `${vaultBaseUrl}/secrets/${secretName}/${secretVersion}?api-version=7.2`;
 const baseUrlVaults = (subscriptionId: string, rg: string, vaultName: string) =>
   `https://management.azure.com/subscriptions/${subscriptionId}/resourceGroups/${rg}/providers/Microsoft.KeyVault/vaults/${vaultName}?api-version=2019-09-01`;
-const baseUrlVaultAddAccessPolicy = (subscriptionId: string, rg: string, vaultName: string) =>
-  `https://management.azure.com/subscriptions/${subscriptionId}/resourceGroups/${rg}/providers/Microsoft.KeyVault/vaults/${vaultName}/accessPolicies/add?api-version=2019-09-01`;
+const baseUrlVaultAddAccessPolicy = (
```
Comment (blackchoey) [inline packages/cli/tests/commonlib/keyVaultValidator.ts:28]: minor: better naming to reflect it's purpose? maybe baseUrlVault`Update`AccessPolicy?

File: packages/cli/tests/e2e/commonUtils.ts [hunk 2]
```diff
@@ -99,7 +99,7 @@ export function getAzureTenantId() {
 }
 
 export function getAzureAccountObjectId() {
-  return cfg.AZURE_ACCOUNT_OBJECT_ID || "c35fa297-dc10-4993-a18c-e8709b8410d1";
+  return cfg.AZURE_ACCOUNT_OBJECT_ID || "";
```
Comment (blackchoey) [inline packages/cli/tests/e2e/commonUtils.ts:102]: should we throw error if no object id is configured?

File: packages/cli/tests/commonlib/keyVaultValidator.ts (hunk 1 above)
Comment (dilin-MS) [reply packages/cli/tests/commonlib/keyVaultValidator.ts:28]: Cool. Will fix that.

Comment (dilin-MS) [reply packages/cli/tests/commonlib/keyVaultValidator.ts:28]: Fixed.

File: packages/cli/tests/e2e/commonUtils.ts (hunk 2 above)
Comment (dilin-MS) [reply packages/cli/tests/e2e/commonUtils.ts:102]: Fixed.

**Review by blackchoey — APPROVED** [review]

**Review by Alive-Fish — APPROVED** [review]

**Pull request merged**